cmd/subctl/root.go

```diff
@@ -35,7 +35,10 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var restConfigProducer = restconfig.NewProducer()
+var (
+	restConfigProducer = restconfig.NewProducer()
+	namespace          string
```

Comment:
This bothers me — we’ve had bugs in the past from global variables which required setup, where code was added which used the variables in code paths where they weren’t set up. I’d rather have a namespace variable specific to each command which can use a namespace (with a shared “namespace flag” function to ensure the help text is consistent); but that’s made complicated by the shared package.

Reply:
Or we can name the vars as `exportNamespace` and `unexportNamespace`?

Reply:
@tpantelis @skitt WDYT?

Reply:
The whole point is to isolate the variables, so we avoid polluting the global namespace; so an `unexportOptions` struct is better than explicitly-named global variables.